# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/findutils-programname.patch
# Copyright (C) 2006 - 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- findutils-4.2.24/gnulib/lib/error.c.orig	2005-05-14 08:03:58.000000000 +0200
+++ findutils-4.2.24/gnulib/lib/error.c	2006-11-02 11:28:06.000000000 +0100
@@ -79,6 +79,11 @@
 
 # include <bits/libc-lock.h>
 
+#elif defined(__UCLIBC__)
+
+# define program_name program_invocation_name
+# include <errno.h>
+
 #else /* not _LIBC */
 
 # if !HAVE_DECL_STRERROR_R && STRERROR_R_CHAR_P
